doc/api/webstreams.md

```diff
@@ -1457,6 +1457,48 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with an `ArrayBuffer` containing the full
   contents of the stream.
 
+```mjs
+import { arrayBuffer } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+const data = await arrayBuffer(readable); // Consumes data stream as an arrayBuffer
+console.log(`from readable: ${data.byteLength}`); // From readable: 42000001
+```
+
+```cjs
+const { arrayBuffer } = require('node:stream/consumers');
+const { Readable } = require('node:stream');
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+async function readFromStream() {
+  const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+  const data = await arrayBuffer(readable); // Consumes data stream as an arrayBuffer
```

Comment:
Also, I'm not sure if those comments are really required here as it'd be self explanatory (not blocking, just an opinion)

Reply:
Yup, thought the same